src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/reference/inputs/EceCharacterReferenceContact.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <v-row>
+    <v-col cols="12" md="12" lg="12" xl="12">
+      <h3>Contact Information</h3>
+      <div role="doc-subtitle">We may contact you to verify or clarify information you provide.</div>
+      <v-row class="mt-5">
+        <v-col cols="12" md="8" lg="6" xl="4">
+          <v-text-field
+            :rules="[Rules.required('Enter your last name')]"
+            label="Last Name"
+            variant="outlined"
+            color="primary"
+            maxlength="100"
+            hide-details="auto"
+            @update:model-value="(value) => $emit('update:model-value', value, { lastName: value })"
+          />
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col cols="12" md="8" lg="6" xl="4">
+          <v-text-field
+            :rules="[Rules.required('Enter your first name')]"
+            label="First Name"
+            variant="outlined"
+            color="primary"
+            maxlength="100"
+            hide-details="auto"
+            @update:model-value="(value) => $emit('update:model-value', value, { firstName: value })"
+          />
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col cols="12" md="8" lg="6" xl="4">
+          <v-text-field
+            :rules="[Rules.required(), Rules.email('Enter your email in the format \'name@email.com\'')]"
+            label="Email"
+            variant="outlined"
+            color="primary"
+            maxlength="200"
+            hide-details="auto"
+            @update:model-value="(value) => $emit('update:model-value', value, { email: value })"
```

Comment:
So are we needing to send both here? 

`value`
and
`{ email: value }`

Reply:
we don't need to necessarily. Value could be '' or null instead of a variable.